src/cf.data.pre

```diff
@@ -4697,13 +4697,16 @@ DOC_START
 	    RESPONSE
 
 		[http::]<Hs	HTTP status code received from the next hop
-		[http::]>Hs	HTTP status code sent to the client
+		[http::]>Hs	HTTP status code of the last sent reply.
 
-		[http::]<h	Reply header. Optional header name argument
-				as for >h
+		[http::]<h	The header of the last sent reply. If that last
+				reply was an interim reply (e.g., HTTP 100 Continue),
+				then the interim reply header is logged.
+				Optional header name argument as for >h.
 
-		[http::]mt	MIME content type
+		[http::]mt	MIME content type of the last sent reply.
```

Comment:
Can we be more specific here? What is a `MIME content type` in this context? The contents of the Content-Type response header? Something more complex?